starter-code/routes/auth.js

```diff
@@ -0,0 +1,116 @@
+const express = require("express");
+const router = express.Router();
+
+router.get("/signup", (req, res, next) => {
+  res.render("auth/signup");
+});
+
+router.get("/login", (req, res, next) => {
+  res.render("auth/login");
+});
+
+router.get("/secret", (req, res, next) => {
+  res.render("secret");
+});
+
+// User model
+const User = require("../models/user");
+
+// BCrypt to encrypt passwords
+const bcrypt = require("bcrypt");
+const bcryptSalt = 10;
+
```

Comment:
It's considered better practice to group all your modules at the beginning in express to make files easier to read, so line 17-21 should be on line 3